services/common/boltzmann/request-handler.js

```diff
@@ -29,7 +29,7 @@ class Context {
     if (this._parsedUrl) {
       return this._parsedUrl;
     }
-    this._parsedUrl = new URL(this.request.url);
+    this._parsedUrl = new URL(this.request.url, 'http://entropic.dev');
```

Comment:
I opted to go with an (inert) URL here because the host value is never used except to keep `new URL()` from throwing an exception; whereas this code is used from a lot of different services which may not _all_ need (or specify) `EXTERNAL_HOST`.